src/makeCalculator.js

```diff
@@ -5,6 +5,47 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, number = 0) {
+      operation.call(this, number);
+
+      return this;
```

Comment:
The `operate` method is correctly using the function shortcut syntax instead of the `function` keyword, which is good practice as per the checklist.